### PR TITLE
Added Comb Sort implementation in Java

### PR DESCRIPTION
**Pull request by Devansu-Yadav**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

Yes.
### Description

(Write your answer here.)

Comb sort is a relatively simple sorting algorithm. It is an improvement over Bubble sort by using a gap of size of more than 1.
### Checklist

- [ ✔️] I've read the contribution guidelines.
- [ ✔️] I've checked the issue list before deciding what to submit.
- [ ✔️] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

#2827

### CODE REVIEW

**Comment by Devansu-Yadav** [comment]
@ankitaggarwal23 @iamrajiv Can u pls review my PR?

File: Java/README.md [hunk 1]
```diff
@@ -203,6 +203,7 @@ _add list here_
 - [Merge Sort Using Recursion](sort/MergeSortRecursion.java)
 - [Shell Sort](sort/ShellSort.java)
 - [Sleep Sort](sort/SleepSort.java)
+- [Comb Sort](sort/CombSort_implementation_in_java.java)
```
Comment (ankitaggarwal23) [inline Java/README.md]: Kindly Renamed `CombSort_implementation_in_java.java` to `CombSort_implementation.java`

File: Java/sort/CombSort_implementation_in_java.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
+/*
+    Comb Sort is a relatively simple sorting algorithm. It is an improvement over Bubble sort
+    by using gap of size more than 1. The gap starts with a large value and starts shrinking by
+    1.3 in every iteration until it reaches the value 1.
+    You can check out more about this Sorting Algorithm here:- https://en.wikipedia.org/wiki/Comb_sort
+*/
+
+import java.util.Scanner;
+
+public class CombSort_implementation_in_java {
```
Comment (ankitaggarwal23) [inline Java/sort/CombSort_implementation_in_java.java]: Kindly Renamed `CombSort_implementation_in_java` to `CombSort_implementation`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Kindly do these changes. Rest looks good to merge.

**Review by iamrajiv — APPROVED** [review]

File: Java/sort/CombSort_implementation_in_java.java (hunk 2 above)
Comment (Devansu-Yadav) [reply Java/sort/CombSort_implementation_in_java.java]: @ankitaggarwal23 Done. Is it fine?

Comment (ankitaggarwal23) [reply Java/sort/CombSort_implementation_in_java.java]: One more left. Just Fix that and We are good to merge this PR. @Devansu-Yadav 
Even In the README.md as well.

Comment (Devansu-Yadav) [reply Java/sort/CombSort_implementation_in_java.java]: @ankitaggarwal23 Done!

Comment (Devansu-Yadav) [reply Java/sort/CombSort_implementation_in_java.java]: @ankitaggarwal23 Done!

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**